packages/lib-classifier/src/components/Classifier/components/Toolbar/Toolbar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Box } from 'grommet'
+import AnnotateButton from './components/AnnotateButton'
+import FieldGuideButton from './components/FieldGuideButton'
+import FullscreenButton from './components/FullscreenButton'
+import MoveButton from './components/MoveButton'
+import ResetButton from './components/ResetButton'
+import RotateButton from './components/RotateButton'
+import ZoomInButton from './components/ZoomInButton'
+import ZoomOutButton from './components/ZoomOutButton'
+
+const toolbarStyles = {
+  border: {
+    color: 'lightGrey',
+    side: 'all'
+  },
+  pad: 'small'
+}
+
+function Toolbar (props) {
```

Comment:
Is this meant to be a generic toolbar, or a toolbar just for images? If it's generic, image functions like zoom, fullscreen and rotate should be split out into an image toolbar.

Reply:
This toolbar is for the image functions. It's worth noting as well that these might move around depending on which layout is chosen, so we may need to do something different in the future (like making this toolbar part of the current layout itself)

Reply:
I'd suggest calling this `ImageToolbar` in that case, leaving room to implement other types of toolbar in future eg. for the TESS data.